docs/developer/guide.md

```diff
@@ -20,3 +21,13 @@ The following steps are needed to introduce a new resource and its respective re
 - Reference the new resource in [pkg/options/resource.go](https://github.com/kubernetes/kube-state-metrics/blob/main/pkg/options/resource.go).
 - Add a sample Kubernetes manifest to be used by tests in the [tests/manifests/](https://github.com/kubernetes/kube-state-metrics/tree/main/tests/manifests) directory.
 - Lastly, and most importantly, actually implement your new resource(s) and its test binary in [internal/store](https://github.com/kubernetes/kube-state-metrics/tree/main/internal/store). Follow the formatting and structure of other resources.
+
+### Add New Metrics
+
+- Make metrics experimental first when introducing them, [Example PR](https://github.com/kubernetes/kube-state-metrics/pull/1910).
+
+| Metric stability level |                    |
+|------------------------|--------------------|
+| EXPERIMENTAL           | basemetrics.ALPHA  |
```

Comment:
Should this stay experimental or should we rename to alpha?

Reply:
I think we can create another issue to discuss whether converting Alpha to Experimental.

From my point of view, Experimental is easier to understand while Alpha aligns with k8s stability level.

Reply:
+1 for `experimental`, since `alpha` can hint toward more stabilty levels (`beta`, etc.) which can be misleading.